git/merge-strategy.md

```diff
@@ -39,33 +39,11 @@ git checkout master
 git merge --no-ff <branch-name> # forces a merge commit and will open commit prompt
 ```
 
-When running the above command you will be prompted for a commit message for your merge. The commit message should be in the following format:
-
-```
-<title of Kanbanery card or summary of feature> [<Task abbreviation> #<Kanbanery task number>] (<Source Control abbreviation> #<Pull Request number>)
-
-<highlights of feature change if any>
-```
-
-And example would look like:
-
-```
-Return 404 on permissions API when no access [KB #1188645] (GH #4)
-
-If a user no access to a study at all we want to be able communicate that to API clients, thus we will return a 404.
-```
-
-#### Abbreviations
-
-|Tasks            |Source Control
-|:----------------|:-------------
-|  KB - Kanbanery |  GH - Github
-|  TR - Trello    |  GL - Gitlab
-|  LK - Lean Kit  |  BB - BitBucket
+When running the above command you will be prompted for a commit message for your merge. The commit message should follow the [Merge Commit Format](merge-commit-format.md).
 
 ### Push the Merged Changes to the Remote
 
-You now have a fully merge repository, but the remote does not.  You need to push those changes now: 
+You now merge complete locally, but the remote does not.  You need to push those changes now:
```

Comment:
This wording feels a little odd as well. Perhaps `Your merge is now completed locally but needs to be updated on the remote repository.` ?